lib/routes/account.js

```diff
@@ -659,6 +659,21 @@ module.exports = function (
               return true
             }
           }
+
+          // If the account was recently created, don't make the user
+          // confirm sign-in for a configurable amount of time. This will reduce
+          // the friction of a user adding a second device.
+          const bypassAccountWithAge = config.signinConfirmation.bypassAccountWithAge
+          const accountCreatedSince = Date.now() - account.createdAt
```

Comment:
At the risk of being a gigantic nit-picker, something about the naming here feels really awkward to me, and past experience has shown us that having clear names for security-related things is important.

Here's an alternative suggestion (*just* a suggestion...):

```
const skipForNewAccounts = config.signinConfirmation.skipForNewAccounts
if (skipForNewAccounts && skipForNewAccounts.enabled) {
  const accountAge = Date.now() - account.createdAt
  if (accountAge <= skipForNewAccounts.maxAge) {
    return true
  }
}
```

And the corresponding config:

```
skipForNewAccounts: {
   enabled :{
       doc: 'Skip sign-in confirmation for newly-created accounts.',
       default: false,
       env: 'SIGNIN_CONFIRMATION_SKIP_FOR_NEW_ACCOUNTS'
   },
   maxAge: {
       doc: 'Maximum age at which an account is considered "new".',
       format: 'duration',
       default: '4 hours',
       env: 'SIGNIN_CONFIRMATION_MAX_AGE_OF_NEW_ACCOUNTS'
   }
}
```

The env var names are still pretty unwieldy though :-/